nomos/data-availability-layer.md

```diff
@@ -0,0 +1,434 @@
+--- 
+title: NOMOS-DATA-AVAILABILITY-PROTOCOL
+name: Nomos Data Availability Protocol
+status: raw
+tags: nomos
+editor: 
+contributors:
+  
+---
+
+## Abstract
+
+This specification describes the protocol for the data availability for the Nomos network.
+Nomos provides several services for network states to create efficient ecosystems.
+Data availabilty is an important problem that network states need to solved. 
+
+## Background
+
+Nomos is a cluster of blockchains known as zones.
+Zones are layer 2 blockchains that utilize Nomos to maintain sovereignty.
+They are initialized by the Nomos network and can utilize Nomos services, but 
+provide resources on their own.
+They can define their own state as they are sovreign networks.
+Nomos provides tools at the global layer that allow zones to define arbitrary configurations.
+Nomos has two global layers offering services to Zones. 
+The base layer provides data availibility guarantees to zones that choose to utilize it. 
+The second layer is the coordination layer which enables state transition verification through zero-knowledge validity proofs. 
+The base layer allows users with resource-limiting devices, also known as light clients, 
+the ability to obtain all block data and process it locally.
+Light clients should be able to access blockchain data similar to a full node.
+To achieve this, 
+the Nomos data availbilty protocol provides guarantees that transaction data within Nomos zones are vaild.
+
+## Motivation and Goal
+
+Decentralized blockchains require full nodes to verify network transactions by downloading all the data of the network.
+This becomes a problem as the blockchain data grows, full nodes will need more resources to download and 
+store the data while maintaining connection to the network.
+Light nodes on the other hand do not download the entire network data because of it's resource limiting nature.
+This retricts the network from scaling as the network is reliant on full nodes to process transactions.
+and requires light nodes to rely on centralized parties. 
+A blockchain can have light nodes prove the validity of transaction data,
+without the light node to downloading all the blockchain data.
```

Comment:
This is not really the focus. Focus is bandwidth reduction. Splitting data into chunks and knowing who to send what really makes an impact as the network grows.

Reply:
@danielSanchezQ Do you mean not the focus for the network or for the role of a DA node in the BL layer for Nomos? Is focus on bandwidth reduction for light nodes, full nodes or both?